scripts/kvm-test.py

```diff
@@ -243,9 +247,18 @@ def build(ctx):
     # --inject-snap $SUBIQUITY_SNAP_PATH
     if project.startswith('subiquity'):
         if ctx.args.quick:
-            with snap_manager('subiquity_test.snap') as snap:
+            with snap_manager('subiquity_new.snap') as snap:
```

Comment:
It turns out that this 'with' statement makes no sense at all, with either the old or new value.  Even using subiquity_new.snap as the value isn't quite right because the path to that snap file is wrong - that snap file is created in tempdir created by the 'quick' script.  I'm pushing a commit that just removes the with statement and I have no idea why this was ever working.